docs/parameter_tables/HomSap/PapuansOutOfAfrica_10J19.csv

```diff
@@ -51,3 +50,5 @@ Migration rate (x10^-4),5.72,(European + East Asian)--Papuan migration rate
 Migration rate (x10^-4),4.42,(European + East Asian)--Ghost migration rate
 Time (yrs.),"37,497",(European + East Asian)--Papuan migration begins
 Time (yrs.),"37,497",(European + East Asian)--Ghost migration begins
+Generation time (yrs.),29,Generation time
+Mutation rate,1.25e-8,Per-base per-generation mutation rate
```

Comment:
where'd you get this, @apragsdale ? Here's what I find: 
![Screenshot from 2021-04-15 16-44-55](https://user-images.githubusercontent.com/1046249/114951947-3b7f6b80-9e0a-11eb-8543-897cbd637450.png)
![Screenshot from 2021-04-15 16-46-02](https://user-images.githubusercontent.com/1046249/114951962-3f12f280-9e0a-11eb-83c0-6ee643ca7498.png)

Reply:
Hmm, I see. Within the implementation, there was already a comment `# mutation_rate = 1.25e-8  # per gen per site, Scally & Durbin 2012`, and I had assumed that it was citing the mutation rate used in the Jacobs et al paper, and I didn't check the paper myself in that case. Good catch, and sorry for the confusion. Looks like the 1.4e-8 value is the correct one here.